content/articles/univariate-time-series-using-recurrent-neural-networks/index.md

```diff
@@ -0,0 +1,356 @@
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Getting started with Recurrent Neural Network (RNN)](#getting-started-with-recurrent-neural-network-rnn)
+- [Monthly milk production dataset](#monthly-milk-production-dataset)
+- [Importing Matplotlib](#importing-matplotlib)
+- [Decomposing the time series into components](#decomposing-the-time-series-into-components)
+- [Splitting the time series dataset into training and testing dates](#splitting-the-time-series-dataset-into-training-and-testing-dates)
+- [Scaling the time series dataset](#scaling-the-time-series-dataset)
+- [Format the scaled data points into batches](#format-the-scaled-data-points-into-batches)
+- [Building the time series model](#building-the-time-series-model)
+- [Compiling the sequential time series model](#compiling-the-sequential-time-series-model)
+- [Fitting the sequential model to the generated batches](#fitting-the-sequential-model-to-the-generated-batches)
+- [Using the trained sequential model to make predictions](#using-the-trained-sequential-model-to-make-predictions)
+- [Plotting the line graph](#plotting-the-line-graph)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Prerequisites]
+- A reader should have the basic knowledge and understanding of the following before implementing the time series model:
+
+- Some basic understanding and knowledge of [time series terms and concepts](/engineering-education/introduction-to-time-series/)
+- A brief introduction to [neural networks architecture and concepts](/engineering-education/introduction-to-neural-networks/)
+- Know how to build an [artificial neural network with Keras](/engineering-education/build-ann-with-keras/)
+- How to run a Python project in [Google Colab](https://colab.research.google.com/)
+
+### Getting started with Recurrent Neural Network (RNN)
+A Recurrent Neural Network is a variant of the typical artificial neural network that can model sequential data. Sequential data depends on historical/past data points to make future predictions.
+
+Recurrent neural networks use internal memory that helps them to store information retrieved from the previous data points and use this information to generate the next data points.
+
+Recurrent neural networks implement a feedback loop within the layers with the hidden layer which distinguishes them from the traditional artificial neural networks.
+
+The image below shows the distinction between a traditional artificial neural network and a recurrent neural network:
+
+![Traditional neural network vs RNN ](/engineering-education/univariate-time-series-using-recurrent-neural-networks/ann-vs-rnn.png)
+
+[Image Source: Simplilearn](https://www.simplilearn.com/ice9/free_resources_article_thumb/Simple_Recurrent_Neural_Network.png)
```

Comment:
```suggestion
![Traditional neural network vs RNN ](/engineering-education/univariate-time-series-using-recurrent-neural-networks/ann-vs-rnn.png)

*[Image Source: Simplilearn](https://www.simplilearn.com/ice9/free_resources_article_thumb/Simple_Recurrent_Neural_Network.png)*
```

Reply:
This is the correct format for adding an image from other sources. Please fix the others too accordingly.